packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -443,10 +443,34 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
     scroll(scrollValue);
   };
 
+  const getFirstVisibleTab = (tabs) => {
+    const containerSize = vertical ? tabsRef.current.clientHeight : tabsRef.current[clientSize];
+    const containerStartBound = Math.round(
+      vertical ? tabsRef.current.scrollTop : tabsRef.current.scrollLeft,
+    );
```

Comment:
```suggestion
    const containerStartBound = Math.round(tabsRef.current[scrollStart]);
```

Similar to the prev line